lbz/authentication.py

```diff
@@ -37,8 +34,6 @@ def get_user_details_from_auth_token(self) -> dict:
         return parsed_user
 
     def _validate_attributes(self, attributes: dict) -> None:
-        if len(attributes) > self._max_attributes:
-            logger.error(
```

Comment:
FYI: This error log would be repeated a level higher when handling a `RuntimeError`.